modules/swagger-codegen/src/main/resources/Java/libraries/okhttp-gson/ApiClient.mustache

```diff
@@ -720,12 +720,13 @@ public class ApiClient {
      *   application/json
      *   application/json; charset=UTF8
      *   APPLICATION/JSON
-     *
+     *   application/vnd.company+json
      * @param mime MIME (Multipurpose Internet Mail Extensions)
      * @return True if the given MIME is JSON, false otherwise.
      */
     public boolean isJsonMime(String mime) {
-        return mime != null && mime.matches("(?i)application\\/json(;.*)?");
+      String jsonMime = "(?i)^(application/json|[^;/ \t]+/[^;/ \t]+[+]json)[ \t]*(;.*)?$";
+      return mime != null && mime.matches(jsonMime);
```

Comment:
Later I will file a PR to put these 2 lines into one with declaring a temporary variable.